h2o-k8s/README.md

```diff
@@ -100,6 +100,9 @@ The name of the application `app: h2o-k8s` must correspond to the name expected
 for the H2O node discovery to work. H2O communicates on port 54321, therefore `containerPort: 54321`must be exposed to
 make it possible for the clients to connect.
 
+There are official H2O Docker images available in Docker hub: `h2oai/h2o-open-source-k8s:<tagname>`. Please make sure to replace the `<tagname>` 
```

Comment:
I would even just say `latest` in the example... and leave a note here about nightly. WDYT?

Reply:
It is nice when examples are good to go for the 99.999% use cases.

Reply:
Done. Also added RAM usage note from Docker hub, as this is important.

Reply:
can you specify a specific release like rel-zahradnik/4?

Reply:
Yes, this is possible. It'll be the `PROJECT_VERSION`, e.g. `3.30.0.5`

Reply:
There are none like that ATM, it's too new.